layouts/partials/header.html

```diff
@@ -437,22 +437,6 @@
     integrity="sha256-7Bdg/UdDMmHDPafpQDYZNPCEbnRP7eAvu+2hEoqRCXs=" crossorigin="anonymous"></script>
   <!-- END: Required by runnable -->
 
-  <script>
-    {{ $currentBranch:= getenv "CURRENT_BRANCH" }}
-    {{ $dgraphEndpoint:= getenv "DGRAPH_ENDPOINT" }}
-
-    window.DGRAPH_ENDPOINT = {{ if (eq $dgraphEndpoint "" ) }}"https://play.dgraph.io/query?latency=true"{{ else }} {{ $dgraphEndpoint }} {{ end }};
```

Comment:
I don't believe these lines (441-444) had anything to do with hotjar, but instead were used for the runnable shortcode. I don't think this is used anywhere in the docs though, but FYI removing these lines don't line up with the description of this PR.

Reply:
Ah, good catch

Reply:
Yup. These lines should be kept. They're for doc version pages and for runnable code.

Example (version): https://dgraph.io/docs/v20.11/dql/
Example (runnable): https://dgraph.io/docs/v20.11/query-language/functions/#term-matching

Reply:
Done